Source/Provers/SMTLib/SMTLibInteractiveTheoremProver.cs

```diff
@@ -223,9 +223,8 @@ public override void FullReset(VCExpressionGenerator generator)
               if (resp.Name != "")
               {
                 usedNamedAssumes.Add(resp.Name);
-                if (libOptions.TrackVerificationCoverage)
-                {
-                  reporter.AddCoveredElement(resp.Name.Substring("aux$$assume$$".Length));
+                if (libOptions.TrackVerificationCoverage) {
+                  reporter.AddCoveredElement(new LabeledElement(resp.Name.Substring("aux$$assume$$".Length)));
```

Comment:
Could you not have called `ParseSolverString` here already?

Reply:
There's an additional bit of string processing going on here beyond what I introduced (and that's been in the Boogie codebase for years). I avoided touching it as part of this PR because I want to more deeply consider how to do so that is both elegant and doesn't break things. Fortunately, I think that it can be done, once I have a deeper understanding of that, without changing anything about the exposed API. This PR is critical because of how it changes the exposed API.

Reply:
Okay